stancon_2020/InferenceData_Stancon2020_cat.ipynb

```diff
@@ -12,33 +12,30 @@
     "\n",
```

Comment:
<p>Should there be two is here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_misc/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>you mean in "ArviZ i InferenceData"?</p>